vector/src/test/java/im/vector/app/features/poll/create/CreatePollViewStates.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022 New Vector Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package im.vector.app.features.poll.create
+
+import im.vector.app.features.poll.PollMode
+import kotlin.random.Random
+
+object CreatePollViewStates {
+
+    const val fakeRoomId = "fakeRoomId"
```

Comment:
For all `const val`, it is preferable to use upper case format like: 

`const val A_FAKE_ROOM_ID = "fakeRoomId"`

Reply:
Right, done.